client/packages/system/src/Patient/CreatePatientModal/PatientResultsTab.tsx

```diff
@@ -154,8 +159,9 @@ export const PatientResultsTab: FC<PatientPanel & { active: boolean }> = ({
   const count = data?.length ?? 0;
 
   useEffect(() => {
+    if (Object.values(searchParams).every(it => it === undefined)) return;
     search(searchParams);
-  }, [patient]);
+  }, [search, searchParams]);
```

Comment:
Don't like how we need to do this "are params empty" check in multiple places but had to fight too many Android build/debug issues that I just did this quick fix for now...